packages/captp/src/captp.js

```diff
@@ -140,6 +140,10 @@ export const makeCapTP = (
     const seen = new Set();
 
     return harden({
+      /**
+       * @param {T} specimen
```

Comment:
![image](https://github.com/endojs/endo/assets/60294/7635ddaa-ddad-4393-ad26-3e481497d3a4)
Is the specimen ready?